UNRELEASED.md

```diff
@@ -1 +1,4 @@
 # Unreleased
+
+**Added:**
+- New `bpk_social_like` icon.
```

Comment:
Shouldn't this appear under a component name?

Reply:
Icons in Android don't belong to a component, they are _[resources](https://developer.android.com/reference/android/content/res/Resources)_.